---
title: "Serratia- and Bacillus-produced Volatiles and their fungistatic properties on Rhizopus stolonifer"
author: Derreck Carter\-House, Ian Wright, Josh Chen, Kerry Mauck, Jason Stajich
abstract: "*Serratia marcescens* lives a lifestyle of diversity and ubiquity. Found nearly everywhere this bacteria is capable of living as a saprobe in soil and water, but also living in community with plants and other rhizobacteria, and in addition as a pathogen to plants, animals and other microbes. Besides being a beautiful example of life’s ability to adapt to survive, Serratia also provides insight to one interesting aspect of microbiology. How do bacteria and fungi communicate?  We have identified multiple strains of Serratia from unique niches and compared their ability to inhibit the growth of common soil inhabitants known to interact with plants, Mucoromycota."
output:
  pdf_document: 
    keep_tex: yes
    toc: yes
    latex_engine: xelatex
editor_options: 
  chunk_output_type: inline
---


```{r setup, include=FALSE, echo=FALSE}
require("knitr")
opts_knit$set(root.dir = "./")
library(readxl)
library(ggplot2)
library(reshape2)
library(VennDiagram)
library(vegan)
library(ggdendro)
library(png)
library(grid)
library(gridExtra)
library(float)
library(kableExtra)
library(ggpubr)
library(plyr)
library(tiff)
options(tinytex.verbose = TRUE)

```

# Colors:
#####  Color Palette by Paletton.com
#####  Palette URL: http://paletton.com/#uid=23E0u0kt+lZlOstrKqzzSiaJidt


*** Primary color:

   shade 0 = #113A73 = rgb( 17, 58,115) = rgba( 17, 58,115,1) = rgb0(0.067,0.227,0.451)
   shade 1 = #385F95 = rgb( 56, 95,149) = rgba( 56, 95,149,1) = rgb0(0.22,0.373,0.584)
   shade 2 = #1D4B8B = rgb( 29, 75,139) = rgba( 29, 75,139,1) = rgb0(0.114,0.294,0.545)
   shade 3 = #072C5F = rgb(  7, 44, 95) = rgba(  7, 44, 95,1) = rgb0(0.027,0.173,0.373)
   shade 4 = #032047 = rgb(  3, 32, 71) = rgba(  3, 32, 71,1) = rgb0(0.012,0.125,0.278)

*** Complement color:

   shade 0 = #AF710B = rgb(175,113, 11) = rgba(175,113, 11,1) = rgb0(0.686,0.443,0.043)
   shade 1 = #E3A948 = rgb(227,169, 72) = rgba(227,169, 72,1) = rgb0(0.89,0.663,0.282)
   shade 2 = #D48F1C = rgb(212,143, 28) = rgba(212,143, 28,1) = rgb0(0.831,0.561,0.11)
   shade 3 = #915A00 = rgb(145, 90,  0) = rgba(145, 90,  0,1) = rgb0(0.569,0.353,0)
   shade 4 = #6B4300 = rgb(107, 67,  0) = rgba(107, 67,  0,1) = rgb0(0.42,0.263,0)


#####  Generated by Paletton.com (c) 2002-2014
```{r colors}
#Divergent:
colors_10=c("#032047","#072C5F","#113A73","#1D4B8B","#385F95","#E3A948","#D48F1C","#AF710B","#915A00","#6B4300")

colors_5blue=c("#032047","#072C5F","#113A73","#1D4B8B","#385F95")
colors_5gold=c("#6B4300","#915A00","#AF710B","#D48F1C","#E3A948")
colors_6div=c("#032047","#113A73","#385F95","#E3A948","#AF710B","#6B4300")
colors_3blue=c("#385F95","#113A73","#032047")
colors_3gold=c("#E3A948", "#AF710B", "#6B4300")
```
\newpage

# Importance: 

This research shows bacteria that live in the soil (Serratia marcescens, Serratia proteamaculans, and Bacillus subtilis) are able to produce compounds (small molecules) that are then volatilized (evaporate into the air) that can travel distances and slow or prevent fungi from growing. A common fungi known as Black Bread Mold, aka Rhizopus stolonifer, can often be found in the soil or rotting strawberries. We have shown in the laboratory that bacteria can stop or slow the growth of this fungi when they share the same air. Each bacteria had many compounds that were volatiles and some were unique to the bacteria that made them. We used purified chemical compounds we found in the bacterial gas and tested them one at a time to see if they work as antifungals. We found that all the tested bacteria produced several compounds that are fungistatic (stops fungal growth). Future work may include looking at the volatiles produced by the bacteria living in the soil or other environments were Rhizopus may come in contact.

# Introduction:

Interactions between Fungi and Bacteria span mutualistic, such as bacteria using fungal hyphae as highways[1], to antagonistic as seen in the famous penicillin discovery [2]. Improvements in imaging and sequencing approaches have aided in illuminating the niches where fungi and bacteria communities coexist, communicate and cooperate. Such as metagenomics in lichen symbioses of cyanobacteria and fungi [3], and what organisms are competition within the topsoil [4], or coexisting in extreme environments [5]. Tools for microbiomics have enabled studies of how fungal and bacterial. For example, Aspergillus nidulans was found to activate secondary metabolites after physical contact with Streptococcus, these metabolites are similar to lecanoric acid found in lichen symbiosis but also functions as a defense compound inhibiting ATP production[6]. Bacteria may help fungi to be better pathogens by creating toxins like Rhizoxin [7]. Bacterial-fungal communities are unique and diverse leading to discoveries from studying these relationships include finding that early lineages of fungi harbor ecto- and endohyphal bacteria and their role in asexual reproduction, nitrogen, and carbon utilization [8–14], mechanisms and applications of bacterial-fungal interactions [15], and microbial volatile bouquets [16] . Bacteria utilize fungal hyphae for high-speed travel, they can eat fungi, and they can consume the metabolites. [1,17–19]. These relationships have yielded many benefits including furthering our understanding of nutrient cycling in ecosystems, fermented foods and beverages, and antibiotics [20].  
Microbial genomes encode genes for many antifungals and antibiotics [21]. Some studies have highlighted antagonism, table 1. Such as work has brought about the approval of Serratia plymuthica as a biocontrol (Rhizostar) that reduces Verticillium wilt by 18.5% and Phytophthora cactorum root rot by 33.4% in strawberry [22]. A conserved response to bacteria was demonstrated when antagonistic Lysobacter enzymogenes was shown to repress fungal response genes with effectors. Another repressor produced by Serratia marcescens (Smar), inhibits the growth of Candida albicans, [23]. Recently, Fusarium fujikuroi and Botrytis cinerea were shown to protect their survival spores after exposure to the bacterial secondary metabolite, Ralstonin, from Ralstonia solanacearum [24]. To add to the complexity, fungal-bacterial interactions occur among communities of organisms where detection and specific response to organisms from plants to animals may be required [15,20]. 


```{r table1, echo=FALSE, message=FALSE, out.width = "80%", warning=FALSE, fig.pos = "H"}
refs <- read_excel("./Data_in/Volatile.xlsx", sheet = "refs")
refs <- as.data.frame(refs)

kable(refs, booktabs= T) %>%
  kable_styling(full_width = T, bootstrap_options = c("striped")) %>%
  column_spec(1,width="8cm")

```

Although fungi can respond to direct contact with bacterial microbe-associated molecular patterns [32], strangely, fungi seem to lack common eukaryotic receptors for bacterial recognition [33]. The lack of receptors may be a result of an effective fungal offense e.g. producing antibiotics, sequestering iron, making their general vicinity inhospitable to bacterial invasion through decreasing pH [32,34]. Due to the fact that microbes may be detrimental to each other, perceiving infochemicals from neighboring competitors or acting on each other from a distance is an important and understudied aspect and additionally, what are the volatiles produced by microbes [35,36].
Bacteria produce volatiles which can diffuse in the soil, and have been shown to be useful communication signals [22,37–39]. The impact of some volatiles can be harmful to organisms, or may serve as infochemicals which fungi may respond to by avoidance, morphology changes, or arresting growth [34]. Some bacteria like Serratia closely interact with plant roots, and form physical barriers barring potential pathogens, in addition, they can provide nutrients and growth hormones like indole-3-acetic acid to the plant [40–43]. Serratia is widely considered a rhizobacteria but how does it interact with the fungal populations nearby?
The fungal growth inhibition by volatiles has been shown in a few systems such as between plants and their pathogens [44] as well as bacteria and fungi (Ralstonia and Fusarium and Botrytis) [25]. Fungal-bacterial communication has been neglected with the lack of tools to analyze the interactions, however, with transcriptomics, genomics, and metabolomics increasing in popularity and decreasing in cost more is being done. Rhizoctonia in direct contact with Serratia plymuthica had transcriptional responses that lead to hyphal morphology changes, defense, and toxin production [34]. The interest in indirect contact, volatiles, and their functions and applications are becoming more popular as well. The chemical diversity of volatiles produced by bacteria is broad with some having antifungal properties [45]. Because volatile profiles differ from bacteria to bacteria a resource called mVOC 2.0 was created showing volatile signatures of many bacteria [46]. A few key volatiles can be a sufficient signature to identity a microbe. In fact, plants identify many volatiles produced by bacteria as a way of communication [47]. Volatiles pass through soil through diffusion and advection [48] which makes volatile communication between microbes effective, analogous to quorum sensing.
Smar is a rhizobacterium and inhabits soils among many environments [49]. Serratia is an effective biocontrol agent due to a number of antimicrobials and enzymes that it secretes. Prodigiosin, a red-pigmented tripyrole compound that has been shown to inhibit mycelial growth and it is secreted it into the environment, in fact, traces of prodigiosin have been found in soil of tea plants [26]. Serratia can produce and secrete enzymes as well such as chitinases that digest fungal cell walls [24,50–52]. Smar has been shown to produce the potent anti-oomycete compound Oocydin which [53]. In addition to this arsonal of products, Serratia volatiles have been identified and shown to affect growth, antibiotic production and gene expression of neighboring bacteria [22,54]. Due to the metabolic cheapness and high vapor pressure, volatiles easily move around in the soil [36]. Most are thought to be waste products of biosynthetic pathways, but some unique volatiles such as γ-butyrolactones can inhibit a wide range of bacteria [55]. Other bacteria have been shown to produce volatiles that can inhibit spore germination, Streptomyces inhibits Cladosporium [56] and Ralstonia produces ralstonins that inhibit germination of Aspergillus flavus [57]. Even fungi can produce volatiles to affect other fungi. Yeast volatiles reduced the ability of Aspergillus to resist insect damage [58], showing that while volatiles may not be fungicidal they can lead to fungal death indirectly. 
One of the volatiles produced by Serratia is 2-undecanone [59], a non-hazardous volatile [59–61] that has fungistatic properties [62,63], efficacy as a mosquito repellent [64], and is used as a fragrance for its minty properties [65]. Undecanoic acid elicits a response in the fungus Trichophyton rubrum leading to up-regulation of transcripts involved in reduction of fatty acid and metabolize 2-undecanone, oxidative stress, and cell wall integrity loss [66]. 

# Materials and Methods: 

Strains: Serratia marcescens (Smar) - Lab strain ADJS-2C_Red [67]), Serratia proteamaculans (Spro) - Lab strain BW106 [1], Bacillus subtilis (Bsub) - Strain E9 [68], Basidiobolus ranarum (Lab strain: AG-B5), Mucor circinelloides (CBS 277.49), Rhizopus stolonifer (NRRL 66455), Alternaria infectoria (Lab strain DB1-7) , and Neurospora crassa (FGSC 4289)
Donut Plates: Luria-Bertani (LB) (10g Peptone, 10g NaCl, 5g Yeast Extract, 7.5g Agar, and 500mL Water) was pipetted into the outer ring of a 100mm petri dish with a 60mm petri dish lid placed inside. Then MEYE or Vogel’s Media was added to the 60mm petri dish lid. 1mL of S.m. was inoculated onto the outer LB ring and allowed to grow for 48 hours at 25 C. Then mycelial plugs or 1000 spores/conidia were added to the media in the central 60mm petri dish lid. Measurements were taken 48 hours after fungi were inoculated. Bsub was previously shown to be effective at inhibiting Fusarium [68] and was used as a positive control
GC-MS: One colony was picked and grown overnight in liquid LB media, then the culture was inoculated onto three plastic Petri dishes and allowed to grow for 48 hours at 28ºC, then volatiles were collected for 6 hours at 25ºC. To serve as positive controls Bsub, Strain E9 was used. To serve as negative controls volatiles from blank Petri dishes and Petri dishes with LB media were collected. 1mL of S.m. grown overnight was added to a 60mm petri dish, headspace volatile collection was taken at 48 hours for 6 hours at 25ºC at .5L/min with air cleaned by a charcoal purification filter pushed into to sample collection apparatus (350 mL Mason Ball jar). Jar lids were fitted with a Teflon liner  and two brass bulkhead unions (pipe thread .64 cm Swagelok, San Diego Valve and fitting Co.) with output into a volatile trap. The volatile trap consisted of ¼ inch glass tube filled with 40 mg HayeSepQ beads 80-100 mesh size.   Volatiles were eluted with 150 ul Dichloromethane spiked with 4ng/ul nonyl-acetate, 2 ng/ul octane and was stored at -80ºC until GC:MS analysis. For GC coupled Mass spect carrier was helium, over ramp was 32 minute 40ºC to 280ºC. DB Wax column, TG-5MS Thermo fisher. Spectra were analyzed with Chromeleon 7 software and compounds were identified based on retention times and RSI (Reverse Match Factor) from the NIST library index. 
Volatile Analysis: Volatile amounts were calculated using area under the curve of a spike control of known mass. The volatiles with a prediction of 85% RSI or higher were kept. Unique volatiles were removed, those only produced by one sample or two samples. The PCA plots and NMDS were produced in R. NMDS was produced with the function metaMDS from the vegan package with the code on Github (https://github.com/Derreckadam/NMDS_Volatiles/tree/master). MetaboAnalyst 4.0 was used to further analyze samples. Samples were scaled, then ANOVA (w/ Tukey’s Post Hoc) identified significantly different volatiles produced between the strains, and the clustering and sample distance was calculated in euclidean space by the Ward.D method. Random forest plot show separation of Bsub from the Serratia strains after just a few generations, but the convergence of the Serratia samples after 200 generations indicating that the two. 
Microscopy: 1000 spores from Rstol were plated onto 8 mL MEYE media with agar [69] in a 60 mm petri dish and allowed to grow for 12 hours at 25ºC. The hyphae were then examined under an inverted compound microscope at 200x. In 2-undecanone treated samples, 1 mg of 2-undecanone was added near the edge of the hyphae on a piece of filter paper and exposed for 1 hour before imaging.
	Pure Volatile Serial Dilutions: 2-undecanone (SIGMA U1303-5ML) was diluted in ethanol in a series of serial dilutions. Each dilution was plated with a volume of 10 µl onto 7.5 mL of Vogel’s media with agar in a 60 mm petri dish. 10 µl of FGSC 4289 conidial suspension of approximately 50 spores/µl was added to each plate. The plates were incubated in 12 hour light and dark conditions at 25ºC, and germinated spores were counted. The EC50 growth curves were calculated with AAT Bioquest, https://www.aatbio.com. 

#Results: 

Results: 
We noticed that when Serratia marcescens (strain ADJS-2C_Red, Smar) streaked on the plate across from fungi, fungal growth was repressed. The growth area could quantitatively be assessed, based on growth with bacteria streaked on either side compared to growth with no bacteria (Figure 1A). However, this method is not ideal for studying volatiles because it allowed for bacterial metabolites or proteins to diffuse through the media, potentially coming in direct contact with the fungal hyphae. Additionally, the bacteria are mobile and could possibly induce a response by direct contact with the fungal hyphae. The “donut” assay  was developed to limit diffusion of bacterial products in the media to fungal hyphae (Figure 1B). This set-up allows for the exchange of gases between the fungi and bacteria, but prevents direct contact and allows for two types of media to be poured (one tailored to the bacteria and the other tailored to the fungi). This is useful for the use of antibiotics in the fungal media or fungi with special growth needs.  We tested the ability of various strains of bacteria and the amount of time they needed to be present on the media before the Rstol were introduced (Figure 2). We tested for this inhibition across many fungi from various phylogenetic clades such as Basidiobolus, Mucor, Rhizopus, Alternaria, and Neurospora. For all strains the exchange of gases was enough to inhibit fungal growth after two days of bacteria growth. LB media was used to grow bacteria and was left uninoculated in the control experiments. The fungal spores/conidia were introduced onto the center plate after 48 hours of bacteria growth. The plates were measured after 24 and 48 hours, (Figure 3). The results show that Spro followed by Smar then Bsub had the highest inhibition of Rstol growth. Spro inhibition of Ncrassa and Rhizopus Smar had an average growth being 1.5 cm at 24 hours and 4 cm at 48 hours. Bsub was a weaker inhibitor of growth with the average growth of the fungi being 2.8 and 5.2 cm at 24 and 48 hours, respectively. Control growth of Rstol with no bacteria inoculated on the LB outer ring was 3.7 cm at 24 hours and had grown to the diameter of the plate (5.7 cm) at 48 hours. 
After establishing that fungal growth inhibitors reside in the volatiles of bacteria we used Gas Chromatography coupled with Mass Spectrometry to identify the compounds produced by our strains of bacteria. Based on previous literature we assumed that we would find 2-undecanone and dimethyl disulfide [30]. On average there were about 50 volatiles produced by each strain. Strains with less than 500ng detected were thrown out, Smar_F and Spro_I, (Figure 4). The volatiles identified are listed in supplemental table 1. To analyze the volatiles, we restricted the results to only the volatiles with a confident prediction (Thermo Mass Spectrometry RSI Score) of over 85%, and volatiles found in more than half of a sample’s replicates, this left us with 5-15 unique volatiles produced per species with 13 compounds produced by all the bacteria (Figure 5). Using an internal standard amounts of volatiles were predicted, this left us with an approximate abundance of compounds. Smar, Spro, and Bsub strain had different volatiles as well as different amounts produced (Figure 6). The top quartile of production had 13 compounds produced by two out of three of the species, only Anisole and 2-methoxyphenol were produced by Smar soley (Figure 7). Non Dimensional Multivariate Scaling (NMDS) shows low stress and resolves Bacillus and Serratia cleanly.
To further analyze the data we used MetaboAnalyst [74,75], an R package for metabolite profiles. Comparisons were made between all three samples, Bsub, Smar and Spro to see if each species could be differentiated by volatile profiles. Auto-scaling centered around the volatile production average was used to make all metabolites comparable (Figure 9), the samples were averaged together and compared to show which samples were differentially produced by the bacterial strains (Figure 10). Principal Component Axis (PCA) analyzed the variance between each of the replicates (Figure 11). ANOVA analysis with a p-value cut off .05 and Tukey’s/HSD analysis reveals eleven significantly produce volatiles compounds (Figure 12, Table, Figure 13). After 5000 trees Spro and Smar were unable to be predicted based on their profiles (Figure 14).  
	The analysis identified volatiles produced by Smar, Spro, and Bsub. We tested 2-undecanone, Dimethyl Trisulfide, Anisole, Lepidine, 2,5-dimethylpyrazine as they were some of the highest produced volatiles from the bacteria. 2-undecanone was chosen because it is a known antifungal [30], is used in food and fragrance, as well as an effective insect repellent approved for use with humans [68,76]. Dimethyl Trisulfide was tested based on the abundance in the Spro profile and sulfur content. Anisole was solely produced by Smar in high amounts. To test the volatiles we loaded the compounds onto a piece of filter paper on a glass microscope slide on the side of a petri dish with media, spores/conidia were inoculated approximately 2 cm away on the media. The effective volatiles for preventing growth were 2-undecanone, Anisole, and Dimethyl Trisulfide, figure (Figure 15). Microscopy phenotype with volatiles reflect the growth inhibition results. Vacualization was found in 2-undecanone and DMTS samples (Figure 16, Microscope Images). 

```{r donut, echo=FALSE, out.height= "90%", fig.pos = "H", fig.cap="A) Streaks of serratia on either side of fungi. B) Donut plate assays of Rstol and other fungi showed the functionality of volatiles inhibiting fungal growth in the presence of bacteria. The bacteria in the LB agar ring on the outside of the plate were not able to come into physical contact with the fungi. Interactions were solely restricted to the volatiles released into the shared headspace of the petri dish. Donut plates both show fungal inhibition of growth with at 48 hours (I E. coli, II Smar, III Smar fungi covered, IV No bacteria)."}
streak <-  rasterGrob(as.raster(readPNG("images/streaks.png")), interpolate = TRUE)
dount <-  rasterGrob(as.raster(readPNG("images/Donuts.png")), interpolate = TRUE)

grid.arrange(streak, dount, ncol = 2)

```

```{r inhibtion_over_days, include=FALSE}
library(cowplot)
#Rhizopus data with on S. pro, S. mar, and B. subtilis

# Get Rhizopus data with many strains (48 Hour incubation)
timelap <- read_excel("./Data_in/Chapter2_data.xlsx", sheet = "Serratia Multistrain Analysis", range = "J1:M19")
timelap$Strain
# Rhizopus name ordering
names.data <- unique(timelap$Strain)
timelap$Strain <- factor(timelap$Strain, levels = names.data)
#boxplot(c ~ Strain, Rhizo, las=2)
names.data

## For loop to obtaim averages and stdv
Strain.unique <- unique(timelap$Strain)
Strain.list <- list()

for (i in 1:length(Strain.unique)){
  # Subset the data frame into each strain
  df.test <- timelap[timelap$Strain %in% Strain.unique[i],]
  # The second element of an apply means: 1 = Rows, 2 = Columns
  # This means we are going to do the mean for all columns in the df
  Strain.list[[i]] <- data.frame(Strain=Strain.unique[i], Time=c("0","24","48"), mean=apply(df.test[,-1],2,mean), sd=apply(df.test[,-1],2,sd))
}

final.data <- as.data.frame(do.call(rbind, Strain.list))
rownames(final.data) <- NULL
#FOR POWERPOINT
#final.data <- subset(final.data, Strain=="E. coli" | Strain=="S. proteamaculans" | Strain=="S. liquefaciens" | Strain=="S. marcescens (ADJS_Red)")
#final.data$Strain <- factor(final.data$Strain,levels = c("E. coli", "S. proteamaculans", "S. liquefaciens", "S. marcescens (ADJS_Red)"))
#final.data$Strain <- factor(final.data$Strain,levels = c("E. coli", "S. proteamaculans", "S. liquefaciens", "S. marcescens (ADJS_Red)"))

Same<-subset(final.data, Time=="0")
oneday<-subset(final.data, Time=="24")
twoday<-subset(final.data, Time=="48")

pd <- position_dodge(1)

timelap.bar.0 <- ggplot(Same, aes(x=Strain, y=mean, fill=Strain)) + geom_bar(stat="identity", position=pd) + geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.1, position=pd) + xlab("") + ylab("Percent Growth Inhibition")  +theme(axis.text.x = element_blank(),legend.position = "none")+ scale_fill_manual(values=colors_6div) + ggtitle("24 Hour Fungal Growth with 24-Hour-Old Bacteria")

timelap.bar.24 <- ggplot(oneday, aes(x=Strain, y=mean, fill=Strain)) + geom_bar(stat="identity", position=pd) + geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.1, position=pd) + xlab("") + ylab("") + ggtitle("24 Hour Fungal Growth with 48-Hour-Old Bacteria") + theme(axis.text.x = element_blank(),legend.position = "none")+scale_fill_manual(values=colors_6div)+ ylab("Percent Growth Inhibition")

timelap.bar.48 <- ggplot(twoday, aes(x=Strain, y=mean, fill=Strain)) + geom_bar(stat="identity", position=pd) + geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.1, position=pd) + xlab("") + ylab("Percent Growth Inhibition") + theme(axis.text.x = element_text(angle=30, hjust=.5, vjust=.5), legend.position = "none")+ scale_fill_manual(values=colors_6div)+ggtitle("24 Hour Fungal Growth with 72-Hour-Old Bacteria")
timelap.bar.48
leg.plot <- ggplot(twoday, aes(x=Strain, y=mean, fill=Strain)) + geom_bar(stat="identity", position=pd) + theme(axis.text.x = element_text(angle = 90, hjust=.5, vjust=1),legend.position = "bottom")+ scale_fill_manual(values=colors_6div)

leg <- get_legend(leg.plot)
plot_grid(timelap.bar.0, timelap.bar.24, timelap.bar.48, nrow=1)

```

```{r timelapse, echo=FALSE, warning=FALSE, out.width = "90%", fig.pos="H",fig.cap="Figure 1: We tested the ability of various strains of bacteria to inhibit fungal growth at 0, 24, and 48 hours"}

top<-plot_grid(timelap.bar.0, timelap.bar.24, timelap.bar.48, nrow=3)
pdf("R-plots/time_assay_ppt.pdf", width = 10, height=4)
plot_grid(timelap.bar.0, timelap.bar.24, timelap.bar.48, leg)
dev.off

pdf("R-plots/time_assay.pdf", width = 6, height=8)
plot_grid(timelap.bar.0, timelap.bar.24, timelap.bar.48,ncol=1, rel_heights = c(.3,.3,.45))
dev.off

```

```{r include=FALSE}
library(ggplot2)
straw <- read_excel("./Data_in/Volatile.xlsx", sheet = "Strawberry")

straw.df<- as.data.frame(straw)

straw.sub<-subset(straw.df, straw.df$State =="Control" | straw.df$State == "Smar")
straw.df$State<- factor(straw.df$State,levels = c("Control", "Smar", "2-Undecanone"))

my_comparisons <- list( c("Control", "Smar"), c("Control", "2-Undecanone"))
stat_compare_means(aes(label = ..p.signif..),
                  method = "t.test", ref.group = "0.5")
straw.plot <- ggplot(straw.df, aes(x=State, y=Spores, fill=State)) + geom_boxplot() +stat_compare_means(aes(label = ..p.signif..), method = "t.test", ref.group = "Control") + xlab("Condition")+  theme(legend.position = "none") + ggtitle("Smar Reduces Spore Production of Rstol") + scale_fill_manual(values=colors_3gold)
straw.plot
#powerpoint
png("./R-plots/strawplot.png", res=200, height=600, width =800)
straw.plot
dev.off()
#dissertation
pdf("./R-plots/strawplot.pdf", height=5, width=5)
straw.plot
dev.off()
```



```{r include=FALSE}
#Pathogens Inhbition
#Fungal data with S. pro, S. mar, and B. subtilis

Mucor<- read_excel("./Data_in/Chapter2_data.xlsx", sheet = "Mucor")
AF<- read_excel("./Data_in/Chapter2_data.xlsx", sheet = "AF")
Rhizo<- read_excel("./Data_in/Chapter2_data.xlsx", sheet = "Rhizo")
Ncra<- read_excel("./Data_in/Chapter2_data.xlsx", sheet = "Ncrassa")
AGB5<- read_excel("./Data_in/Chapter2_data.xlsx", sheet = "Basidiobolus")
Conidi<- read_excel("./Data_in/Chapter2_data.xlsx", sheet = "Conidi")
Actino<- read_excel("./Data_in/Chapter2_data.xlsx", sheet = "Actino")
Alter<- read_excel("./Data_in/Chapter2_data.xlsx", sheet = "Alter")

AGB5.m <- melt(as.data.frame(AGB5), id.vars = "Condition")
colnames(AGB5.m) <- c("Strain","Time","Growth")
AGB5.mse <- ddply(AGB5.m, c("Strain","Time") , summarise, N = length(Growth), mean = mean(Growth), sd   = sd(Growth), se   = sd / sqrt(N))
AGB5.mse 

AGB5.plot <- ggplot(AGB5.mse, aes(x=Time, y=mean, group=Strain, fill=Strain))+ ggtitle(expression(italic("Basidiobolus ranarum"))) +geom_bar(stat="identity", position=pd)+ theme(legend.position = "none") + geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.1, position=pd)+ylab("Percent Petri Dish")+ xlab("") + scale_fill_manual(limits = c("Blank", "Bsub", "Smar", "Spro"), values=c("#D48F1C", "#915A00", "#385F95", "#1D4B8B"))

Conidi.m <- melt(as.data.frame(Conidi), id.vars = "Condition")
colnames(Conidi.m) <- c("Strain","Time","Growth")
Conidi.mse <- ddply(Conidi.m, c("Strain","Time") , summarise, N = length(Growth), mean = mean(Growth), sd   = sd(Growth), se   = sd / sqrt(N))

Conidi.plot <- ggplot(Conidi.mse, aes(x=Time, y=mean, group=Strain, fill=Strain)) + ggtitle(expression(italic("Conidiobolus rhyzosporus"))) +geom_bar(stat="identity", position=pd)+ theme(legend.position = "none") + geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.1, position=pd)+ylab("Percent Petri Dish")+ xlab("") + scale_fill_manual(limits = c("Blank", "Bsub", "Smar", "Spro"), values=c("#D48F1C", "#915A00", "#385F95", "#1D4B8B"))

Actino.m <- melt(as.data.frame(Actino), id.vars = "Condition")
colnames(Actino.m) <- c("Strain","Time","Growth")
Actino.mse <- ddply(Actino.m, c("Strain","Time") , summarise, N = length(Growth), mean = mean(Growth), sd   = sd(Growth), se   = sd / sqrt(N))

Actino.plot <- ggplot(Actino.mse, aes(x=Time, y=mean, group=Strain, fill=Strain))  + ggtitle(expression(italic("Actinomucor elegans"))) +geom_bar(stat="identity", position=pd)+ theme(legend.position = "none") + geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.1, position=pd)+ylab("Percent Petri Dish")+ xlab("") + scale_fill_manual(limits = c("Blank", "Bsub", "Smar", "Spro"), values=c("#D48F1C", "#915A00", "#385F95", "#1D4B8B"))

Alter.m <- melt(as.data.frame(Alter), id.vars = "Condition")
colnames(Alter.m) <- c("Strain","Time","Growth")
Alter.mse <- ddply(Alter.m, c("Strain","Time") , summarise, N = length(Growth), mean = mean(Growth), sd   = sd(Growth), se   = sd / sqrt(N))

Alter.plot <- ggplot(Alter.mse, aes(x=Time, y=mean, group=Strain, fill=Strain))  + ggtitle(expression(italic("Alternaria infectoria"))) +geom_bar(stat="identity", position=pd)+ theme(legend.position = "none") + geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.1, position=pd)+ylab("Percent Petri Dish")+ xlab("") + scale_fill_manual(limits = c("Blank", "Bsub", "Smar", "Spro"), values=c("#D48F1C", "#915A00", "#385F95", "#1D4B8B"))

#M circenelloides
Mucor.m <- melt(as.data.frame(Mucor), id.vars = "Condition")
colnames(Mucor.m) <- c("Strain","Time","Growth")
Mucor.mse <- ddply(Mucor.m, c("Strain","Time") , summarise, N = length(Growth), mean = mean(Growth), sd   = sd(Growth), se   = sd / sqrt(N))

Mucor.plot <- ggplot(Mucor.mse, aes(x=Time, y=mean, group=Strain, fill=Strain)) + ggtitle(expression(italic("Mucor circinelloides"))) +geom_bar(stat="identity", position=pd)+ theme(legend.position = "none") + geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.1, position=pd)+ylab("Percent Petri Dish")+ xlab("") + scale_fill_manual(limits = c("Blank", "Bsub", "Smar", "Spro"), values=c("#D48F1C", "#915A00", "#385F95", "#1D4B8B"))



#Aspergillus
AF.m <- melt(as.data.frame(AF), id.vars = "Condition")
colnames(AF.m) <- c("Strain","Time","Growth")
AF.mse <- ddply(AF.m, c("Strain","Time") , summarise, N = length(Growth), mean = mean(Growth), sd   = sd(Growth), se   = sd / sqrt(N))

AF.plot <- ggplot(AF.mse, aes(x=Time, y=mean, group=Strain, fill=Strain)) + ggtitle(expression(italic("Aspergillus fumigatus"))) +geom_bar(stat="identity", position=pd)+ theme(legend.position = "none") + geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.1, position=pd)+ylab("Percent Petri Dish")+ xlab("") + scale_fill_manual(limits = c("Blank", "Bsub", "Smar", "Spro"), values=c("#D48F1C", "#915A00", "#385F95", "#1D4B8B"))

#Rhizo
Rhizo.m <- melt(as.data.frame(Rhizo), id.vars = "Condition")
colnames(Rhizo.m) <- c("Strain","Time","Growth")
Rhizo.mse <- ddply(Rhizo.m, c("Strain","Time") , summarise, N = length(Growth), mean = mean(Growth), sd   = sd(Growth), se   = sd / sqrt(N))

rhizo.plot <- ggplot(Rhizo.mse, aes(x=Time, y=mean, group=Strain, fill=Strain)) + ggtitle(expression(italic("Rhizopus stolonifer"))) +geom_bar(stat="identity", position=pd)+ theme(legend.position = "none") + geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.1, position=pd)+ylab("Percent Petri Dish")+ xlab("") + scale_fill_manual(limits = c("Blank", "Bsub", "Smar", "Spro"), values=c("#D48F1C", "#915A00", "#385F95", "#1D4B8B"))

#Ncrassa
Ncra.m <- melt(as.data.frame(Ncra), id.vars = "Condition")
colnames(Ncra.m) <- c("Strain","Time","Growth")

Ncra.mse <- ddply(Ncra.m, c("Strain","Time") , summarise, N = length(Growth), mean = mean(Growth), sd   = sd(Growth), se   = sd / sqrt(N))

ncrassa.plot <- ggplot(Ncra.mse, aes(x=Time, y=mean, group=Strain, fill=Strain))+ ggtitle(expression(italic("Neurospora crassa"))) +geom_bar(stat="identity", position=pd)+ theme(legend.position = "none") + geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.1, position=pd)+ylab("Percent Petri Dish")+ xlab("") + scale_fill_manual(limits = c("Blank", "Bsub", "Smar", "Spro"), values=c("#D48F1C", "#915A00", "#385F95", "#1D4B8B"))


leg.plot <- ggplot(Ncra.mse, aes(x=Time, y=mean, group=Strain, fill=Strain))+ ggtitle(expression(italic("Neurospora crassa"))) +geom_bar(stat="identity", position=pd)+ theme(legend.position = "bottom") + geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.1, position=pd)+ylab("Percent Petri Dish")+ xlab("") + scale_fill_manual(limits = c("Blank", "Bsub", "Smar", "Spro"), values=c("#D48F1C", "#915A00", "#385F95", "#1D4B8B"))

leg <- get_legend(leg.plot)

png("./R-plots/NcrassavBac.png", width = 900, height=600, res=300)
ncrassa.plot
dev.off()

png("./R-plots/RhizovBac.png", width = 900, height=600, res=300)
rhizo.plot
dev.off()

png("./R-plots/AFvBac.png", width = 900, height=600, res=300)
AF.plot
dev.off()

png("./R-plots/MucorvBac.png", width = 900, height=600, res=300)
Mucor.plot
dev.off()

png("./R-plots/AltervBac.png", width = 900, height=600, res=300)
Alter.plot+ylab("Percent Petri Dish")
dev.off()

png("./R-plots/ActinovBac.png", width = 900, height=600, res=300)
Actino.plot
dev.off()

png("./R-plots/AGB5vBac.png", width = 900, height=600, res=300)
AGB5.plot
dev.off()

png("./R-plots/ConidivBac.png", width = 900, height=600, res=300)
Conidi.plot
dev.off()

png("./R-plots/legvBac.png", width = 900, height=600, res=300)
leg.plot
dev.off()


```

```{r pathos, echo=FALSE, warning=FALSE, out.width = "90%", fig.pos="H",fig.cap="Pathogenic fungi and their growth inhibtion with bacterial volatiles"}
library(cowplot)
fungi_grid <- plot_grid(ncrassa.plot, rhizo.plot, Conidi.plot, Mucor.plot, AF.plot, Alter.plot,AGB5.plot, Actino.plot, labels = c('A', 'B', 'C', 'D', 'E', 'F', 'G', 'H'), ncol = 2)
plot_grid(fungi_grid, leg, ncol = 1, rel_heights = c(1, .1))
pdf("./R-plots/OtherFungi.pdf", height = 10, width = 6)
plot_grid(fungi_grid, leg, ncol = 1, rel_heights = c(1, .1))
dev.off()

```

```{r perecent_inhibtion, echo=FALSE}
library(readxl)
inhib <- read_excel("Data_in/Chapter2_data.xlsx", sheet = "Donut_percentinhibtion", range = "A44:E68")

stdev <- read_excel("Data_in/Chapter2_data.xlsx", sheet = "Donut_percentinhibtion", range = "A70:E94")

inhib.m <- melt(inhib, id.vars=c("Fungi", "Time"))
stdev.m <- melt(stdev, id.vars=c("Fungi", "Time"))

# 24 Hours
inhib.24 <- subset(inhib.m, Time=="24 Hours")
stdev.24 <- subset(stdev.m, Time=="24 Hours")
Funginames<-unique(inhib.24$Fungi)
myplots.24 <- vector('list', length(Funginames)) 

for (i in Funginames){
    message(i)
    myplots.24[[i]] <- local({
        i <- i
        inhib.24.sub <- subset(inhib.24, Fungi==i)
        stdev.24.sub <- subset(stdev.24, Fungi==i)
        Title<-as.character(i)
        p1 <- ggplot(inhib.24.sub, aes(inhib.24.sub$variable, inhib.24.sub$value, fill=variable)) + geom_bar(stat="identity", position = position_dodge()) + geom_errorbar(aes( ymin=inhib.24.sub$value+stdev.24.sub$value, ymax=inhib.24.sub$value-stdev.24.sub$value), position = position_dodge()) + ylab("Percent Inhibtion") + theme(legend.position = "NULL", axis.text.y = element_blank(), axis.ticks.x = element_blank(), axis.title.x = element_blank()) + scale_fill_manual(values=colors_3blue) + coord_flip() + xlab("") +labs(title=bquote(italic(.(Title)))) 
        print(p1)
    })
}

leg.plot <- ggplot(inhib.24.sub, aes(inhib.24.sub$variable, inhib.24.sub$value, fill=variable)) + geom_bar(stat="identity", position = position_dodge())+ labs(fill="Bacteria")+ scale_fill_manual(values=colors_3blue)
leg <- get_legend(leg.plot)

PercentInhibtion24hours<-plot_grid(myplots.24[["Neurospora crassa"]], myplots.24[["Aspergillus fumigatus"]], myplots.24[["Alternaria infectoria"]], myplots.24[["Rhizopus stolonifer"]], myplots.24[["Mucor circinelloides"]], myplots.24[["Actinomucor elegans"]], myplots.24[["Conidiobolus rhizosporus"]], myplots.24[["Basidiobolus ranarum"]], leg, labels = c("*","","*","*","*","","*(Sp)",""),
  label_size = 10,
  label_x = .91, label_y = .18,
  hjust = 1, vjust = -0.5)
PercentInhibtion24hours

#48 Hours
inhib.48 <- subset(inhib.m, Time=="48 Hours")
stdev.48 <- subset(stdev.m, Time=="48 Hours")
Funginames<-unique(inhib.48$Fungi)
myplots.48 <- vector('list', length(Funginames)) 

for (i in Funginames){
    message(i)
    myplots.48[[i]] <- local({
        i <- i
        inhib.48.sub <- subset(inhib.48, Fungi==i)
        stdev.48.sub <- subset(stdev.48, Fungi==i)
        Title<-as.character(i)
        p1 <- ggplot(inhib.48.sub, aes(inhib.48.sub$variable, inhib.48.sub$value, fill=variable)) + geom_bar(stat="identity", position = position_dodge()) + geom_errorbar(aes( ymin=inhib.48.sub$value+stdev.48.sub$value, ymax=inhib.48.sub$value-stdev.48.sub$value), position = position_dodge()) + ylab("Percent Inhibtion") + theme(legend.position = "NULL", axis.text.y = element_blank(), axis.ticks.x = element_blank(), axis.title.x = element_blank()) + scale_fill_manual(values=colors_3blue) + coord_flip() + xlab("") +labs(title=bquote(italic(.(Title)))) 
        print(p1)
    })
}

leg.plot <- ggplot(inhib.48, aes(inhib.48$variable, inhib.48$value, fill=variable)) + geom_bar(stat="identity", position = position_dodge())+ labs(fill="Bacteria")+ scale_fill_manual(values=colors_3blue)
leg <- get_legend(leg.plot)

PercentInhibtion48hours<-plot_grid(myplots.48[["Neurospora crassa"]], myplots.48[["Aspergillus fumigatus"]], myplots.48[["Alternaria infectoria"]], myplots.48[["Rhizopus stolonifer"]], myplots.48[["Mucor circinelloides"]], myplots.48[["Actinomucor elegans"]], myplots.48[["Conidiobolus rhizosporus"]], myplots.48[["Basidiobolus ranarum"]], leg, labels = c("*","","*(Sp)","*(Sp/Sm)","*(Sp/Sm)","","*(Sp)","*"),
  label_size = 10,
  label_x = .91, label_y = .2,
  hjust = 1, vjust = -0.5)


#72 Hours
inhib.72 <- subset(inhib.m, Time=="72 Hours")
stdev.72 <- subset(stdev.m, Time=="72 Hours")
Funginames<-unique(inhib.72$Fungi)
myplots.72 <- vector('list', length(Funginames)) 

for (i in Funginames){
    message(i)
    myplots.72[[i]] <- local({
        i <- i
        inhib.72.sub <- subset(inhib.72, Fungi==i)
        stdev.72.sub <- subset(stdev.72, Fungi==i)
        Title<-as.character(i)
        p1 <- ggplot(inhib.72.sub, aes(inhib.72.sub$variable, inhib.72.sub$value, fill=variable)) + geom_bar(stat="identity", position = position_dodge()) + geom_errorbar(aes( ymin=inhib.72.sub$value+stdev.72.sub$value, ymax=inhib.72.sub$value-stdev.72.sub$value), position = position_dodge()) + ylab("Percent Inhibtion") + theme(legend.position = "NULL", axis.text.y = element_blank(), axis.ticks.x = element_blank(), axis.title.x = element_blank()) + scale_fill_manual(values=colors_3blue) + coord_flip() + xlab("") +labs(title=bquote(italic(.(Title)))) 
        print(p1)
    })
}

leg.plot <- ggplot(inhib.72, aes(inhib.72$variable, inhib.72$value, fill=variable)) + geom_bar(stat="identity", position = position_dodge())+ labs(fill="Bacteria")+ scale_fill_manual(values=colors_3blue)
leg <- get_legend(leg.plot)

PercentInhibtion72hours<-plot_grid(myplots.72[["Neurospora crassa"]], myplots.72[["Aspergillus fumigatus"]], myplots.72[["Alternaria infectoria"]], myplots.72[["Rhizopus stolonifer"]], myplots.72[["Mucor circinelloides"]], myplots.72[["Actinomucor elegans"]], myplots.72[["Conidiobolus rhizosporus"]], myplots.72[["Basidiobolus ranarum"]], leg, labels = c("*(Sp)","*","","","*(Sp)","","*(Sp)","*(Sm)"),
  label_size = 10,
  label_x = .7, label_y = .2,
  hjust = -0.5, vjust = -0.5)

  
pdf("./R-plots/PercentInhibtion24hours.pdf", width=8, height = 8)
PercentInhibtion24hours
dev.off()

pdf("./R-plots/PercentInhibtion48hours.pdf", width=8, height = 8)
PercentInhibtion48hours
dev.off()

pdf("./R-plots/PercentInhibtion72hours.pdf", width=8, height = 8)
PercentInhibtion72hours
dev.off()


```

```{r include=FALSE}

Samples <- read_excel("./Data_in/Chapter2_data.xlsx", sheet = "Total_production")

Samples.df <- as.data.frame(Samples)

rownames(Samples.df) <- Samples$`Sample`

vol_total <- ggplot(Samples.df, aes(x=Samples.df$`Sample`, y=Samples.df$`Detected Volatiles (ng)`, fill=Samples.df$`Detected Volatiles (ng)`)) + geom_bar(stat="identity") + guides(fill=FALSE) + theme(axis.text.x = element_text(angle = 90, hjust = 1)) + ylab("ng of Volatiles") + xlab("Sample") + scale_fill_gradient2(low="#113A73", mid="#AF710B", high="#E3A948", midpoint = 1000) + geom_hline(yintercept=200, linetype="dotted")
vol_total

```

```{r totes, echo=FALSE, warning=FALSE, out.width = "90%", fig.pos="H",fig.cap="Effect of the most abundance VOCs produced by bacteria on Rhizopus growth after 24 hours"}
pdf("./R-plots/totalGCMS.pdf", heigh=5, width=8)
vol_total
dev.off()
```

```{r volatable, echo=FALSE, out.width = "90%", fig.pos = "H", fig.cap="Table of differentially produced volatiles"}
library(reshape2)

#Table
volatable <- read_excel("./Data_in/Chapter2_data.xlsx", sheet = "VolaTable")
volatable.df <- as.data.frame(volatable)
volatable.m <- melt(volatable.df)
volatable.sub <- subset(volatable.m, volatable.m$value > 0)
rownames(volatable.sub) <- NULL
write.csv(volatable.sub, "./R-plots/volatable.csv", row.names = FALSE, col.names = c("Compound", "Sample", "Mass Detected (ng)"))


#Figure
compds <- read_excel("./Data_in/Chapter2_data.xlsx", sheet = "Most Abundant", range = "E72:H134")
colnames(compds)
compds.sub<-subset(compds, Bsub>100 | Smar>100 | Spro>100)
compds.sub
compds.m <- reshape2::melt(compds.sub)
library(scales)
compds.plot<-ggplot(compds.m, aes(reorder(Compounds, -value), value, fill=variable)) + geom_bar(stat="identity", position = position_dodge()) + theme(axis.text.x = element_text(angle=90, hjust=1, vjust=.1)) + ylab("Nanograms detected (log10)") + xlab("") + scale_fill_manual(values = c("#D48F1C", "#385F95", "#072C5F")) + scale_y_log10(breaks = trans_breaks("log10", function(x) 10^x),
              labels = trans_format("log10", math_format(10^.x))) + ggtitle("DMTS, Anisole, and Butanoic Acid are Highly Produced") + labs(fill="Producing Bacteria")+ coord_flip() 


compds.bsub <- subset(compds.m, variable=="Bsub"& value>50)

compds.bsub.plot<- ggplot(compds.bsub, aes(x=reorder(compds.bsub$Compounds, -compds.bsub$value), y=value, fill=variable)) + geom_bar(stat="identity", position = position_dodge()) + theme(axis.text.x = element_text(angle=90, hjust=1, vjust=.1), legend.position = "none") + ylab("Nanograms detected") + xlab("")+ coord_flip()+ggtitle("Bsub Volatiles")+ scale_y_log10()

compds.smar <- subset(compds.m, variable=="Smar"& value>50)

compds.smar.plot<-ggplot(compds.smar, aes(x=reorder(compds.smar$Compounds, -compds.smar$value), y=value, fill=variable)) + geom_bar(stat="identity", position = position_dodge()) + theme(axis.text.x = element_text(angle=90, hjust=1, vjust=.1), legend.position = "none") + ylab("Nanograms detected") + xlab("")+ coord_flip()+ggtitle("Smar Volatiles")+ scale_y_log10()

compds.spro <- subset(compds.m, variable=="Spro" & value>50)

compds.spro.plot<-ggplot(compds.spro, aes(x=reorder(compds.spro$Compounds, -compds.spro$value), y=value, fill=variable)) + geom_bar(stat="identity", position = position_dodge()) + theme(axis.text.x = element_text(angle=90, hjust=1, vjust=.1), legend.position = "none") + ylab("Nanograms detected") + xlab("") + coord_flip() +ggtitle("Spro Volatiles")+ scale_y_log10()

compds.spro.plot
library(kableExtra)

#webshot::install_phantomjs()
volatable.sub %>%
  kable(col.names = c("Compound", "Sample", "Mass Detected (ng)")) %>%
  kable_styling(bootstrap_options = c("striped", "hover"))%>%
  save_kable("./R-plots/volatable.pdf")

pdf("./R-plots/abund_compds.pdf", height = 8, width = 7)
compds.plot
dev.off()

```


```{r venndia, echo=FALSE,fig.pos="H", out.width = "90%",fig.cap="Over lap of related volatiles produced by three species of bacteria"}


grid.newpage()
  Venn <- draw.triple.venn(area1 = 14, area2 = 16, area3 = 11, n12 = 3, n23 = 9, n13 = 2, n123 = 2, category = c("Bsub", "Smar", "Spro"), lty = rep("solid", 3), fill = c("#D48F1C", "#385F95", "#072C5F"), scaled = TRUE, euler.d = TRUE)



pdf("./R-plots/Venn.pdf", height = 4, width = 4)

grid.newpage()
  Venn <- draw.triple.venn(area1 = 14, area2 = 16, area3 = 11, n12 = 3, n23 = 9, n13 = 2, n123 = 2, category = c("Bsub", "Smar", "Spro"), lty = rep("solid", 3), fill = c("#D48F1C", "#385F95", "#072C5F"), scaled = TRUE, euler.d = TRUE)
  
dev.off()


png("./R-plots/Venn.png", height = 400, width = 500, res=150)

grid.newpage()
  Venn <- draw.triple.venn(area1 = 14, area2 = 16, area3 = 11, n12 = 3, n23 = 9, n13 = 2, n123 = 2, category = c("Bsub", "Smar", "Spro"), lty = rep("solid", 3), fill =c("#56B4E9", "#E69F00", "#999999"))
  
dev.off()

```


```{r include=FALSE}
#NMDS Plot
library(vegan)
volatiles <- read_excel("./Data_in/Chapter2_data.xlsx", sheet = "NMDS_1")
volatiles_names <- read_excel("./Data_in/Chapter2_data.xlsx", sheet = "NMDS_2")
volatiles
rownames(volatiles) <- volatiles_names$Sample

volatiles.matrix=as.matrix(volatiles)

vol.rel <- decostand(volatiles.matrix, method = "total")
vol.rel_distmat <- vegdist(vol.rel, method = "bray")
vol.rel_distmat <- as.matrix(vol.rel_distmat, labels = T)
write.csv(vol.rel_distmat, "volatiles_relative_distance_matrix_nmds.csv")

vol.rel_NMS <- metaMDS(vol.rel_distmat, distance = "bray", k = 2, maxit = 999, trymax = 500, wascores = TRUE)

goodness(vol.rel_NMS)
stressplot(vol.rel_NMS)
plot(vol.rel_NMS)

``` 

```{r nmds, echo=FALSE, out.width = "90%", fig.pos="H",fig.cap="NMDS showing differences between Bacillus and Serratia"}

treat=c(rep("Bsub",6),rep("Smar",6),rep("Spro",8))
ordiplot(vol.rel_NMS,type="points")
ordihull(vol.rel_NMS,groups=treat,draw="polygon",col=c("#D48F1C", "skyblue", "#072C5F"),label=F, scaling = 3, main = expression(air == 2), cex = 2)
legend(x="bottomleft", legend=unique(treat), col=c("#D48F1C", "skyblue", "#072C5F"), pch=16)

pdf("./R-plots/NMDS.pdf", height=5, width=4)
treat=c(rep("Bsub",6),rep("Smar",6),rep("Spro",8))
ordiplot(vol.rel_NMS,type="points")
ordihull(vol.rel_NMS,groups=treat,draw="polygon",col=c("#D48F1C", "skyblue", "#072C5F"),label=F, scaling = 3, main = expression(air == 2), cex = 2)
legend(x="bottomleft", legend=unique(treat), col=c("#D48F1C", "skyblue", "#072C5F"), pch=16)
dev.off()

```

```{r metabolanalyst_nooutliers, echo=FALSE, include=FALSE}

metanr_packages <- function(){
  
metr_pkgs <- c("Rserve", "ellipse", "scatterplot3d", "Cairo", "randomForest", "caTools", "e1071", "som", "impute", "pcaMethods", "RJSONIO", "ROCR", "globaltest", "GlobalAncova", "Rgraphviz", "preprocessCore", "genefilter", "pheatmap", "SSPA", "sva", "Rcpp", "pROC", "data.table", "limma", "car", "fitdistrplus", "lars", "Hmisc", "magrittr", "methods", "xtable", "pls", "caret", "lattice", "igraph", "gplots", "KEGGgraph", "reshape", "RColorBrewer", "tibble", "siggenes", "plotly", "xcms", "CAMERA", "fgsea", "MSnbase", "BiocParallel", "metap", "reshape2", "scales")
  
  list_installed <- installed.packages()
  
  new_pkgs <- subset(metr_pkgs, !(metr_pkgs %in% list_installed[, "Package"]))
  
  if(length(new_pkgs)!=0){
    
    if (!requireNamespace("BiocManager", quietly = TRUE))
      install.packages("BiocManager")
    BiocManager::install(new_pkgs, version = "3.8")
    print(c(new_pkgs, " packages added..."))
  }
  
  if((length(new_pkgs)<1)){
    print("No new packages added...")
  }
}

metanr_packages()

if (!requireNamespace("BiocManager", quietly = TRUE))
  install.packages("BiocManager")
#BiocManager::install(c("plotly", "texi2dvi", "impute", "pcaMethods", "globaltest", "GlobalAncova", "Rgraphviz", "preprocessCore", "genefilter", "SSPA", "sva", "limma", "KEGGgraph", "siggenes", "xcms", "CAMERA", "fgsea", "MSnbase", "BiocParallel"))
#devtools::install_cran("latexpdf")
#install.packages("devtools")
#library(devtools)

#devtools::install_github("xia-lab/MetaboAnalystR", build = TRUE, build_opts = c("--no-resave-data", "--no-manual"))

library(MetaboAnalystR)

mSet<-InitDataObjects("conc", "stat", FALSE)
mSet<-Read.TextData(mSet, "./MetaboAnalysis_Data//MetaboAnalyst_data_removed_outliers.csv", "rowu", "disc");
mSet<-SanityCheckData(mSet)
mSet<-ReplaceMin(mSet);
mSet<-PreparePrenormData(mSet)
mSet<-Normalization(mSet, "NULL", "NULL", "AutoNorm", ratio=FALSE, ratioNum=20)
mSet<-PlotNormSummary(mSet, "norm_2_", "png", 600, width=NA)
mSet<-PlotSampleNormSummary(mSet, "snorm_2_", "png", 600, width=NA)
mSet<-ANOVA.Anal(mSet, F, 0.05, "fisher", FALSE)
mSet<-PlotANOVA(mSet, "aov_2_", "png", 600, width=NA)
mSet<-ANOVA.Anal(mSet, F, 0.05, "tukey", FALSE)
mSet<-PlotANOVA(mSet, "aov_3_", "png", 600, width=NA)
und<-PlotCmpdView(mSet, "2-Undecanone", "png", 72, width=NA)
dmts<-PlotCmpdView(mSet, "Dimethyl trisulfide", "png", 72, width=NA)
ani<-PlotCmpdView(mSet, "Anisole", "png", 72, width=NA)
mSet<-PCA.Anal(mSet)
mSet<-PlotPCAPairSummary(mSet, "pca_pair_2_", "png", 600, width=NA, 5)
mSet<-PlotPCAScree(mSet, "pca_scree_2_", "png", 600, width=NA, 5)
mSet<-PlotPCA2DScore(mSet, "pca_score2d_2_", "png", 600, width=NA, 1,2,0.95,0,0)
mSet<-PlotPCALoading(mSet, "pca_loading_2_", "png", 600, width=NA, 1,2);
mSet<-PlotPCABiplot(mSet, "pca_biplot_2_", "png", 600, width=NA, 1,2)
mSet<-PlotPCA3DScoreImg(mSet, "pca_score3d_2_", "png", 600, width=NA, 1,2,3, 40)
#mSet<-PlotPCA3DLoading(mSet, "pca_loading3d_2_", "json", 1,2,3)
mSet<-PlotHeatMap(mSet, "heatmap_1_", "png", 72, width=NA, "norm", "row", "euclidean", "ward.D","bwm", "overview", T, T, NA, T, F)
#mSet<-PlotSubHeatMap(mSet, "heatmap_3_", "png", 600, width=NA, "norm", "row", "euclidean", "ward.D","bwm", "tanova", 58, "overview", T, T, T, F)
#mSet<-PlotSubHeatMap(mSet, "heatmap_3_", "png", 600, width=NA, "norm", "row", "correlation", "ward.D","bwm", "tanova", 58, "overview", T, T, T, F)

mSet<-PlotSubHeatMap(mSet, "heatmap_3_", "png", 600, width=NA, "norm", "row", "correlation", "average","bwy", "tanova", 63, "overview", T, T, T, F)
mSet<-PlotHeatMap(mSet, "heatmap_2_", "png", 600, width=NA, "norm", "row", "correlation", "average","bwm", "overview", T, T, NA, T, F)
mSet<-RF.Anal(mSet, 500,7,1)
mSet<-PlotRF.Classify(mSet, "rf_cls_2_", "png", 72, width=NA)
mSet<-PlotRF.VIP(mSet, "rf_imp_2_", "png", 72, width=NA)
mSet<-PlotRF.Outlier(mSet, "rf_outlier_2_", "png", 72, width=NA)
mSet<-RF.Anal(mSet, 5000,13,1)
mSet<-PlotRF.Classify(mSet, "rf_cls_3_", "png", 600, width=NA)
mSet<-PlotRF.VIP(mSet, "rf_imp_3_", "png", 72, width=NA)
mSet<-PlotRF.Outlier(mSet, "rf_outlier_3_", "png", 600, width=NA)
mSet<-PlotHCTree(mSet, "tree_2_", "png", 600, width=NA, "spearman", "average")

img1 <-  rasterGrob(as.raster(readPNG("./Dimethyl trisulfide_dpi72.png")), interpolate = FALSE)
img2 <-  rasterGrob(as.raster(readPNG("./2-Undecanone_dpi72.png")), interpolate = FALSE)
img3 <-  rasterGrob(as.raster(readPNG("./Anisole_dpi72.png")), interpolate = FALSE)

png("./R-plots/difexprvol.png", width = 960)
grid.arrange(img1, img2, img3, nrow = 1)
dev.off
```


```{r pca, echo=FALSE, out.width = "90%", fig.pos="H",fig.cap="PCA plot showing variation amount samples of Bsub, Smar, and Spro"}

knitr::include_graphics("pca_score3d_2_dpi600.png")
```

```{r meta_heat, echo=FALSE, out.width = "90%", fig.pos="H", fig.cap="Shows autoscaled volatiles and the production relative to each sample"}
knitr::include_graphics("./heatmap_3_dpi600.png")
```

```{r meta_heat_ave, echo=FALSE, out.width = "90%", fig.pos="H", fig.cap="Average of volatiles produced by each species of bacteria"}
knitr::include_graphics("./R-plots/heatmap_7_dpi72.png")
```

```{r asdf, echo=FALSE, out.width = "90%", fig.pos="H", fig.cap="ANOVA results showing 95% confidence or higher 11 compounds differentially produced by Bacteria"}
knitr::include_graphics("./R-plots/aov_2_dpi600.png")
```

```{r anovatable, echo=FALSE, out.width = "90%", fig.pos = "H", fig.cap="Table of differentially produced volatiles"}

anov_res <- read_excel("./Data_in/Chapter2_data.xlsx", sheet = "ANOVA")
anov_res <- as.data.frame(anov_res)

library(kableExtra)
#webshot::install_phantomjs()
table_vol <- anov_res %>%
  kable() %>%
  kable_styling(bootstrap_options = c("striped", "hover"))%>%
  save_kable("./R-plots/ANOVA-table.pdf")


```


```{r anova_compounds, echo=FALSE, out.width = "100%", fig.pos="H", fig.cap="Relationship of differentially produced volatiles between each bacteria species"}
grid.arrange(img1, img2, img3, nrow = 3)

```


```{r include=FALSE}
# Pure volatile inhibition
#Get pure volatile assay data
library(ggpubr)
abund <- read_excel("./Data_in/Chapter2_data.xlsx", sheet = "Abundant_VOC_RS_24")
abund$Compound <- factor(abund$Compound, levels = c("Dimethyl trisulfide","Anisole", "2-Undecanone", "2,5-Dimethylpyrazine",  "Lepidine", "5-Methyl-2-furylmethanol","Tropone","Unexposed"))
abund
abund.plot<-ggplot(abund, aes(x=Compound, y=Diameter, fill=Compound)) + geom_boxplot() + ylab("Diameter of Rstol mycelium (cm)") + xlab("10 mg of Compound") +   guides(fill=FALSE)+scale_fill_manual(values=colors_10)+annotate("text", y=3, x=c("Dimethyl trisulfide","Anisole", "2-Undecanone", "2,5-Dimethylpyrazine",  "Lepidine", "5-Methyl-2-furylmethanol"),label = "**") + coord_flip()
abund.plot


```

```{r abundance, echo=FALSE, out.width = "90%", fig.pos="H",fig.cap="Growth measurements after 24 hours of exposure to 10mg of each compound"}
abund.plot
pdf("./R-plots/vol_mycelia.pdf", height = 5, width =4)
abund.plot 
dev.off()
```


```{r microscopy, echo=FALSE, out.width="90%", fig.cap="Panel of Microscope images"}

micro_con <- rasterGrob(as.raster(readPNG("images/40x_con_6.png")), interpolate = FALSE)
micro_dmpz <-  rasterGrob(as.raster(readPNG("images/40x_dmpz_7.png")), interpolate = FALSE)
micro_und <-  rasterGrob(as.raster(readPNG("images/40x_und_5.png")), interpolate = FALSE)
micro_an <-  rasterGrob(as.raster(readPNG("images/40x_An_1.png")), interpolate = FALSE)
micro_dmts <-  rasterGrob(as.raster(readPNG("images/40x_DMTS_7.png")), interpolate = FALSE)

library(cowplot)

title = cowplot::ggdraw() + cowplot::draw_label("40X "~italic(Rhizopus)~ " Hyphae Exposed to Volatiles", size = 20)
top_row = cowplot::plot_grid(micro_con, micro_dmpz, micro_und, labels = c('A', 'B', 'C'), ncol=3)
bottom_row = cowplot::plot_grid(micro_an, micro_dmts, labels = c('D', 'E'), ncol=2)

plot_grid(top_row, bottom_row, ncol=1)

png("R-plots/Micro.png")
plot_grid(top_row, bottom_row, ncol=1)
dev.off()


micro_nc_con <- rasterGrob(as.raster(readPNG("./images/NC_Control.png")), interpolate = FALSE)
micro_nc_und <-  rasterGrob(as.raster(readPNG("./images/NC_und.png")), interpolate = FALSE)
plot_grid(micro_nc_con, micro_nc_und, labels = c('A', 'B'), ncol=1)

png("./R-plots/NC_40x.png")
plot_grid(micro_nc_con, micro_nc_und, labels = c('A', 'B'), ncol=1)
dev.off()
```

```{r include=FALSE}
straw <- read_excel("./Data_in/Volatile.xlsx", sheet = "Strawberry")

straw.df<- as.data.frame(straw)

straw.sub<-subset(straw.df, straw.df$State =="Control" | straw.df$State == "2-Undecanone")
straw.sub

straw.plot2 <- ggplot(straw.sub, aes(x=State, y=Spores, fill=State)) + geom_boxplot() + stat_compare_means(method = "t.test", label.x = c("2-Undecanone")) + xlab("Condition") + theme(legend.position = "none") + ggtitle("2-Undecanone Reduces Rstol Spore Production On Strawberry")

```

```{r strawberry, echo=FALSE, out.width="90%", fig.cap="Spores recovered from strawberries innoculated with Rstol in the presence of S. marcescens"}
straw.plot2
png("R-plots/StrawSpore2.png")
straw.plot2
dev.off()
```


```{r genus_volatiles, include=FALSE}

genus_gc <- read_excel("./Data_in/Chapter2_data.xlsx", sheet = "Comparative_Genus")
genus_gc

# Create dendro
rownames(genus_gc) <- genus_gc$Strain

gc.dendro <- as.dendrogram(hclust(d = dist(x = genus_gc)))
dendro.plot <- ggdendrogram(data = gc.dendro, rotate = TRUE)

# Preview the plot
dendro.plot

#make heatmap

gc.long <- melt(genus_gc, id = c("Strain", "Genus"))

#ggplot(genus_gc.m, aes(Samples, variable, fill=value)) + geom_tile()


heatmap.plot <- ggplot(data = gc.long, aes(x = Strain, y = variable)) + geom_point(alpha=.7, aes(color=Genus, size=as.character(gc.long$value)), shape=20) + ylab("Compound") + xlab("Bacteria strains") + theme(axis.text.x = element_text(angle = 90, hjust = 1)) + labs(fill = "Probability of Presence")+ scale_size_manual(values=c(0,5)) + guides(size = FALSE)

heatmap.plot
pdf("./R-plots/Genus_volatiles.pdf", width = 8, height =10)
heatmap.plot
dev.off()

#Order Branches
branch.order <- order.dendrogram(gc.dendro)
gc.long$Samples <- factor(x = gc.long$Strain,
                                levels = unique(gc.long$Strain[branch.order]), ordered = TRUE)


#make sorted heatmap
gc.long
heatmap.plot.sorted <- ggplot(data = gc.long, aes(x = Strain, y = variable)) +
  geom_tile(aes(fill = value)) +
  scale_fill_gradient(low = "grey", high = "blue", na.value = "black") +  ylab("Compound") +
  xlab("List of Strains") +
  theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
  labs(fill = "Probability of Presence")

pdf("Genus_volatiles.pdf", height = 16, width = 16)
heatmap.plot.sorted
dev.off()
```

# Discussion:

The biggest impact on the field. A reexamination of volatiles produced by Serratia marcescens reveals several antifungal compounds. An issue with 2-undecanone is that it has a very short halflife of 1.2 days [76].
The context of impact within the field - Many studies are beginning to incorporate microbial communities into their work, the volatile profiles of these communities are at times neglected but can clearly play a major role. 
Support from other work includes microbial community assembly work is being done, this adds to the body of work showing the importance of fungal bacteria interplay [1].
Limitations of the work are that we are limited to the bacterial profile being created while bacteria are on LB media, hardly a realistic environmental conditions. It would be interesting to see the volatiles produced in various soil types and with various plant partners to see if there is a core volatome. 

# Conclusion:

It appears the key to resisting the bacteria compounds is to metabolize or neutralize the volatiles. More work needs to be done in other systems, especially outside of a petri dish. Spores/Conidia can sense the presence of volatiles and wait until favorable conditions, this would be an interesting follow-up experiment to learn about environmental cues that stop fungi from germinating from conidia. 
Looking into additional compounds to find more fungal responses or if there is a conserved fungal response. Anisole would be an interesting compound to follow up on, only found in Serratia marcescens (ADJS-Red). Not in Bacillus. 
Searching for new antifungals is important. Looking to the future, volatile profiles with multiple antifungal components may be important for the war against fungi. 



# References


1. 	Zhang Y, Kastman EK, Guasto JS, Wolfe BE. Fungal networks shape dynamics of bacterial dispersal and community assembly in cheese rind microbiomes. Nat Commun. 2018;9: 336.
2. 	Fleming A. On the Antibacterial Action of Cultures of a Penicillium, with Special Reference to their Use in the Isolation of B. influenzæ. Br J Exp Pathol. 1929;10: 226.
3. 	Meiser A, Otte J, Schmitt I, Grande FD. Sequencing genomes from mixed DNA samples - evaluating the metagenome skimming approach in lichenized fungi. Sci Rep. 2017;7: 14881.
4. 	Bahram M, Hildebrand F, Forslund SK, Anderson JL, Soudzilovskaia NA, Bodegom PM, et al. Structure and function of the global topsoil microbiome. Nature. 2018;560: 233–237.
5. 	Gonzalez E, Pitre FE, Pagé AP, Marleau J, Guidi Nissim W, St-Arnaud M, et al. Trees, fungi and bacteria: tripartite metatranscriptomics of a root microbiome responding to soil contamination. Microbiome. 2018;6: 53.
6. 	Schroeckh V, Scherlach K, Nützmann H-W, Shelest E, Schmidt-Heck W, Schuemann J, et al. Intimate bacterial-fungal interaction triggers biosynthesis of archetypal polyketides in Aspergillus nidulans. Proc Natl Acad Sci U S A. 2009;106: 14558–14563.
7. 	Partida-Martinez LP, Hertweck C. Pathogenic fungus harbours endosymbiotic bacteria for toxin production. Nature. 2005;437: 884–888.
8. 	Uehling J, Gryganskyi A, Hameed K, Tschaplinski T, Misztal PK, Wu S, et al. Comparative genomics of Mortierella elongata and its bacterial endosymbiont Mycoavidus cysteinexigens. Environ Microbiol. 2017; doi:10.1111/1462-2920.13669
9. 	Li Z, Yao Q, Dearth SP, Entler MR, Castro Gonzalez HF, Uehling JK, et al. Integrated proteomics and metabolomics suggests symbiotic metabolism and multimodal regulation in a fungal-endobacterial system. Environ Microbiol. 2017;19: 1041–1053.
10. Desirò A, Hao Z, Liber JA, Benucci GMN, Lowry D, Roberson R, et al. Mycoplasma-related endobacteria within Mortierellomycotina fungi: diversity, distribution and functional insights into their lifestyle. ISME J. 2018;12: 1743–1757.
11. Hoffman MT, Arnold AE. Diverse bacteria inhabit living hyphae of phylogenetically diverse fungal endophytes. Appl Environ Microbiol. 2010;76: 4063–4075.
12. Partida-Martinez LP, Monajembashi S, Greulich K-O, Hertweck C. Endosymbiont-dependent host reproduction maintains bacterial-fungal mutualism. Curr Biol. 2007;17: 773–777.
13. Shaffer JP, U’Ren JM, Gallery RE, Baltrus DA, Arnold AE. An Endohyphal Bacterium (Chitinophaga, Bacteroidetes) Alters Carbon Source Use by Fusarium keratoplasticum (F. solani Species Complex, Nectriaceae). Front Microbiol. 2017;8: 350.
14. Pawlowska TE, Gaspar ML, Lastovetsky OA, Mondo SJ, Real-Ramirez I, Shakya E, et al. Biology of Fungi and Their Bacterial Endosymbionts. Annu Rev Phytopathol. 2018;56: 289–309.
15. Deveau A, Bonito G, Uehling J, Paoletti M, Becker M, Bindschedler S, et al. Bacterial - Fungal Interactions: ecology, mechanisms and challenges. FEMS Microbiol Rev. 2018; doi:10.1093/femsre/fuy008
16. Misztal PK, Lymperopoulou DS, Adams RI, Scott RA, Lindow SE, Bruns T, et al. Emission Factors of Microbial Volatile Organic Compounds from Environmental Bacteria and Fungi. Environ Sci Technol. 2018;52: 8272–8282.
17. Nazir R, Warmink JA, Boersma H, van Elsas JD. Mechanisms that promote bacterial fitness in fungal-affected soil microhabitats. FEMS Microbiol Ecol. 2010;71: 169–185.
18. Nazir R, Mazurier S, Yang P, Lemanceau P, van Elsas JD. The Ecological Role of Type Three Secretion Systems in the Interaction of Bacteria with Fungi in Soil and Related Habitats Is Diverse and Context-Dependent. Front Microbiol. 2017;8: 38.
19. Warmink JA, Nazir R, Corten B, van Elsas JD. Hitchhikers on the fungal highway: The helper effect for bacterial migration via fungal hyphae. Soil Biol Biochem. 2011/4;43: 760–765.
20. Frey-Klett P, Burlinson P, Deveau A, Barret M, Tarkka M, Sarniguet A. Bacterial-fungal interactions: hyphens between agricultural, clinical, environmental, and food microbiologists. Microbiol Mol Biol Rev. 2011;75: 583–609.
21. Garbeva P, Hordijk C, Gerards S, de Boer W. Volatile-mediated interactions between phylogenetically different soil bacteria. Front Microbiol. 2014;5: 289.
22. Kurze S, Bahl H, Dahl R, Berg G. Biological Control of Fungal Strawberry Diseases by Serratia plymuthica HRO-C48. Plant Dis. 2001;85: 529–534.
23. Trunk K, Peltier J, Liu Y-C, Dill BD, Walker L, Gow NAR, et al. The type VI secretion system deploys antifungal effectors against microbial competitors. Nature Microbiology. 2018; 1.
24. Spraker JE, Wiemann P, Baccile JA, Venkatesh N, Schumacher J, Schroeder FC, et al. Conserved Responses in a War of Small Molecules between a Plant-Pathogenic Bacterium and Fungi. MBio. 2018;9. doi:10.1128/mBio.00820-18
25. Mannaa M, Oh JY, Kim KD. Biocontrol Activity of Volatile-Producing Bacillus megaterium and Pseudomonas protegens against Aspergillus flavus and Aflatoxin Production on Stored Rice Grains. Mycobiology. 2017;45: 213–219.
26. Gao H, Li P, Xu X, Zeng Q, Guan W. Research on Volatile Organic Compounds From Bacillus subtilis CF-3: Biocontrol Effects on Fruit Fungal Pathogens and Dynamic Changes During Fermentation. Front Microbiol. 2018;9: 456.
27. Machado AP, Anzai M, Fischman O. Bacillus subtilis induces morphological changes in Fonsecaea pedrosoi in vitro resulting in more resistant fungal forms in vivo. J Venom Anim Toxins Incl Trop Dis. 2010;16: 592–598.
28. Minerdi D, Bossi S, Gullino ML, Garibaldi A. Volatile organic compounds: a potential direct long-distance mechanism for antagonistic action of Fusarium oxysporum strain MSA 35. Environ Microbiol. 2009;11: 844–854.
29. Vespermann A, Kai M, Piechulla B. Rhizobacterial volatiles affect the growth of fungi and Arabidopsis thaliana. Appl Environ Microbiol. 2007;73: 5639–5641.
30. Popova AA, Koksharova OA, Lipasova VA, Zaitseva JV, Katkova-Zhukotskaya OA, Eremina SI, et al. Inhibitory and Toxic Effects of Volatiles Emitted by Strains of Pseudomonas and Serratia on Growth and Survival of Selected Microorganisms, Caenorhabditis elegans, and Drosophila melanogaster. Biomed Res Int. 2014;2014. doi:10.1155/2014/125704
31. Gkarmiri K, Finlay RD, Alström S, Thomas E, Cubeta MA, Högberg N. Transcriptomic changes in the plant pathogenic fungus Rhizoctonia solani AG-3 in response to the antagonistic bacteria Serratia proteamaculans and Serratia plymuthica. BMC Genomics. 2015;16: 630.
32. Woodhams DC, LaBumbard BC, Barnhart KL, Becker MH, Bletz MC, Escobar LA, et al. Prodigiosin, Violacein, and Volatile Organic Compounds Produced by Widespread Cutaneous Bacteria of Amphibians Can Inhibit Two Batrachochytrium Fungal Pathogens. Microb Ecol. 2017; doi:10.1007/s00248-017-1095-7
33. Using Serratia plymuthica to control fungal pathogens of plants. In: Loop [Internet]. [cited 25 Jun 2016]. Available: http://loop.frontiersin.org/publications/29877540
34. Schmidt R, Jager V de, Zühlke D, Wolff C, Bernhardt J, Cankar K, et al. Fungal volatile compounds induce production of the secondary metabolite Sodorifen in Serratia plymuthica PRI-2C. Sci Rep. 2017;7: 862.
35. Srobel G, Li JY, Sugawara F, Koshino H, Harper J, Hess WM. Oocydin A, a chlorinated macrocyclic lactone with potent anti-oomycete activity from Serratia marcescens. Microbiology. 1999;145 ( Pt 12): 3557–3564.
36. Dhar Purkayastha G, Mangar P, Saha A, Saha D. Evaluation of the biocontrol efficacy of a Serratia marcescens strain indigenous to tea rhizosphere for the management of root rot disease in tea. PLoS One. 2018;13: e0191761.
37. Hu W, Kuang F, Lu Z, Zhang N, Chen T. Killing Effects of an Isolated Serratia marcescens KH-001 on Diaphorina citri via Lowering the Endosymbiont Numbers. Front Microbiol. 2018;9: 860.
38. Duzhak AB, Panfilova ZI, Duzhak TG, Vasyunina EA, Shternshis MV. Role of prodigiosin and chitinases in antagonistic activity of the bacterium Serratia marcescens against the fungus Didymella applanata. Biochemistry . 2012;77: 910–916.
39. Someya N, Nakajima M, Watanabe K, Hibi T, Akutsu K. Potential of Serratia marcescens strain B2 for biological control of rice sheath blight. Biocontrol Sci Technol. 2005;15: 105–109.
40. Ipcho S, Sundelin T, Erbs G, Kistler HC, Newman M-A, Olsson S. Fungal Innate Immunity Induced by Bacterial Microbe-Associated Molecular Patterns (MAMPs). G3 . 2016;6: 1585–1595.
41. Soanes DM, Talbot NJ. Comparative Genome Analysis Reveals an Absence of Leucine-Rich Repeat Pattern-Recognition Receptor Proteins in the Kingdom Fungi. PLoS One. 2010;5: e12725.
42. Wheatley RE. The consequences of volatile organic compound mediated bacterial and fungal interactions. Antonie Van Leeuwenhoek. 2002;81: 357–364.
43. Schmidt R, Cordovez V, de Boer W, Raaijmakers J, Garbeva P. Volatile affairs in microbial interactions. ISME J. 2015;9: 2329–2335.
44. Schulz-Bohm K, Martín-Sánchez L, Garbeva P. Microbial Volatiles: Small Molecules with an Important Role in Intra- and Inter-Kingdom Interactions. Front Microbiol. 2017;8: 2484.
45. Ossowicki A, Jafra S, Garbeva P. The antimicrobial volatile power of the rhizospheric isolate Pseudomonas donghuensis P482. PLoS One. 2017;12: e0174362.
46. Schmidt R, Etalo DW, de Jager V, Gerards S, Zweers H, de Boer W, et al. Microbial Small Talk: Volatiles in Fungal-Bacterial Interactions. Front Microbiol. 2015;6: 1495.
47. Neupane S, Finlay RD, Alström S, Elfstrand M, Högberg N. Transcriptional responses of the bacterial antagonist Serratia plymuthica to the fungal phytopathogen Rhizoctonia solani. Environ Microbiol Rep. 2015;7: 123–127.
48. Kai M, Crespo E, Cristescu SM, Harren FJM, Francke W, Piechulla B. Serratia odorifera: analysis of volatile emission and biological impact of volatile compounds on Arabidopsis thaliana. Appl Microbiol Biotechnol. 2010;88: 965–976.
49. Hoffman MT, Gunatilaka MK, Wijeratne K, Gunatilaka L, Arnold AE. Endohyphal bacterium enhances production of indole-3-acetic acid by a foliar fungal endophyte. PLoS One. 2013;8: e73132.
50. Taghavi T, Kim C, Rahemi A. Role of Natural Volatiles and Essential Oils in Extending Shelf Life and Controlling Postharvest Microorganisms of Small Fruits. Microorganisms. 2018;6. doi:10.3390/microorganisms6040104
51. Kanchiswamy CN, Malnoy M, Maffei ME. Chemical diversity of microbial volatiles and their potential for plant growth and productivity. Front Plant Sci. 2015;6: 151.
52. Lemfack MC, Gohlke B-O, Toguem SMT, Preissner S, Piechulla B, Preissner R. mVOC 2.0: a database of microbial volatiles. Nucleic Acids Res. 2018;46: D1261–D1265.
53. Sharifi R, Ryu C-M. Sniffing bacterial volatile compounds for healthier plants. Curr Opin Plant Biol. 2018;44: 88–97.
54. Minnich M. Behavior and determination of volatile organic compounds in soil: A literature review [Internet]. Lockheed Environmental Systems and Technologies Co., Las Vegas, NV (United States); 1993 May. Report No.: PB-94-100153/XAB. Available: https://www.osti.gov/biblio/5582173
55. Ashelford KE, Fry JC, Bailey MJ, Day MJ. Characterization of Serratia isolates from soil, ecological implications and transfer of Serratia proteamaculans subsp. quinovora Grimont et al. 1983 to Serratia quinivorans corrig., sp. nov. Int J Syst Evol Microbiol. 2002;52: 2281–2289.
56. De Vleesschauwer D, Höfte M. Using Serratia plymuthica to control fungal pathogens of plants. CAB Reviews. 2003; Available: https://biblio.ugent.be/publication/385377/file/582805
57. Gutiérrez-Román MI, Holguín-Meléndez F, Bello-Mendoza R, Guillén-Navarro K, Dunn MF, Huerta-Palacios G. Production of prodigiosin and chitinases by tropical Serratia marcescens strains with potential to control plant pathogens. World J Microbiol Biotechnol. 2012;28: 145–153.
58. Weise T, Thürmer A, Brady S, Kai M, Daniel R, Gottschalk G, et al. VOC emission of various Serratia species and isolates and genome analysis of Serratia plymuthica 4Rx13. FEMS Microbiol Lett. 2014;352: 45–53.
59. Schulz S, Dickschat JS, Kunze B, Wagner-Dobler I, Diestel R, Sasse F. Biological activity of volatiles from marine and terrestrial bacteria. Mar Drugs. 2010;8: 2976–2987.
60. Herrington PR, Craig JT, Chea CY, Sheridan JE. Inhibition of spore germination by volatiles from Streptomyces griseoruber. Soil Biol Biochem. 1985;17. Available: http://agris.fao.org/agris-search/search.do?recordID=US201301443033
61. Khalid S, Baccile JA, Spraker JE, Tannous J, Imran M, Schroeder FC, et al. NRPS-Derived Isoquinolines and Lipopetides Mediate Antagonism between Plant Pathogenic Fungi and Bacteria. ACS Chem Biol. 2018;13: 171–179.
62. Caballero Ortiz S, Trienens M, Pfohl K, Karlovsky P, Holighaus G, Rohlfs M. Phenotypic responses to microbial volatiles render a mold fungus more susceptible to insect damage. Ecol Evol. 2018;8: 4328–4339.
63. Plyuta V, Lipasova V, Popova A, Koksharova O, Kuznetsov A, Szegedi E, et al. Influence of volatile organic compounds emitted by Pseudomonas and Serratia strains on Agrobacterium tumefaciens biofilms. APMIS. 2016;124: 586–594.
64. Yu K, Hamilton-Kemp TR, Archbold DD, Collins RW, Newman MC. Volatile compounds from Escherichia coli O157:H7 and their absorption by strawberry fruit. J Agric Food Chem. 2000;48: 413–417.
65. Groenhagen U, Baumgartner R, Bailly A, Gardiner A, Eberl L, Schulz S, et al. Production of bioactive volatiles by different Burkholderia ambifaria strains. J Chem Ecol. 2013;39: 892–906.
66. Reddy DN, Al-Rajab AJ. Chemical composition, antibacterial and antifungal activities of Ruta graveolens L. volatile oils. Cogent Chemistry. 2016;2: 1220055.
67. Giorgio A, De Stradis A, Lo Cantore P, Iacobellis NS. Biocide effects of volatile organic compounds produced by potential biocontrol rhizobacteria on Sclerotinia sclerotiorum. Front Microbiol. 2015;6: 1056.
68. Bohbot JD, Dickens JC. Insect repellents: modulators of mosquito odorant receptor activity. PLoS One. 2010;5: e12138.
69. Khan IA, Abourashed EA. Leung’s Encyclopedia of Common Natural Ingredients: Used in Food, Drugs and Cosmetics. Wiley; 2009.
70. Mendes NS, Bitencourt TA, Sanches PR, Silva-Rocha R, Martinez-Rossi NM, Rossi A. Transcriptome-wide survey of gene expression changes and alternative splicing in Trichophyton rubrum in response to undecanoic acid. Sci Rep. 2018;8: 2520.
71. Aryal SK, Carter-House D, Stajich JE, Dillman AR. Microbial associates of the southern mole cricket (Scapteriscus borellii) are highly pathogenic. J Invertebr Pathol. 2017;150: 54–62.
72. Na F. Identification of Two Novel Fungal Species Associated with Kuroshio Shot Hole Borer (Euwallacea sp.) and Evaluation of Novel Biological Control Method to Inhibit the Fungal Associates of the Invasive Ambrosia Beetle Species in California [Internet]. UC Riverside. 2016. Available: https://escholarship.org/uc/item/8394m1k0
73. Benny GL. Methods Used by Dr. R. K. Benjamin, and Other Mycologists, to Isolate Zygomycetes. Aliso: A Journal of Systematic and Evolutionary Botany. 2008;26: 37–61.
74. Chong J, Yamamoto M, Xia J. MetaboAnalystR 2.0: From Raw Spectra to Biological Insights. Metabolites. 2019;9. doi:10.3390/metabo9030057
75. Chong J, Xia J. MetaboAnalystR: an R package for flexible and reproducible analysis of metabolomics data. Bioinformatics. 2018;34: 4313–4314.
76. TOXNET. In: NIH Toxicology Data Network [Internet]. 30 Aug 2006 [cited 6 Sep 2019]. Available: https://toxnet.nlm.nih.gov/cgi-bin/sis/search2/r?dbs+hsdb:@term+@rn+@rel+112-12-9


